ibm/service/kubernetes/resource_ibm_container_vpc_cluster.go

```diff
@@ -322,6 +322,14 @@ func ResourceIBMContainerVpcCluster() *schema.Resource {
 				RequiredWith:     []string{"kms_instance_id", "crk"},
 			},
 
+			"cluster_security_groups": {
```

Comment:
can we rename this argument `security_groups` because the resource already indicates its a cluster

Reply:
sure, I renamed it